ppl/zqd/recruiter/recruitworker.go

```diff
@@ -12,10 +12,13 @@ import (
 	"go.uber.org/zap"
 )
 
-func RecruitWorkers(ctx *proc.Context, workerCount int, recruiter string, workerstr string) ([]string, error) {
-	if workerstr != "" {
-		// Special case: workerstr is used for ZTests
-		workers := strings.Split(workerstr, ",")
+// RecruitWorkers is used by the zqd root process to recruit workers for a distributed query.
+func RecruitWorkers(ctx *proc.Context, workerCount int, conf WorkerConfig, logger *zap.Logger) ([]string, error) {
+	logger.Info("recruit workers", zap.Int("count", workerCount))
```

Comment:
Log strings should start with a capital letter.

Reply:
Oops, I got the convention for error messages and log messages mixed up. I will capitalize all the log messages.